test/parallel/test-util-deprecate-invalid-code.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+const common = require("../common");
+const util = require("util");
+
+[1, true, false, null, {}].forEach(notString => {
+  common.expectsError(
+    () => {
+      util.deprecate(() => {}, "message", 123);
+    },
+    {
+      code: "ERR_INVALID_ARG_TYPE",
+      type: global.TypeError,
+      message: `The "code" argument must be of type string`
```

Comment:
This can be a plain single-quoted string instead of a template string.

Reply:
Done